converter.go

```diff
@@ -85,26 +90,30 @@ func convertCoordinates(lon, lat float64) (float64, float64) {
 /*
 Iterate over each coordinate and reassign coordinates to converted values
 */
-func coordinateIterator(buildings *Buildings) {
+func coordinateIterator(buildings *geojson.FeatureCollection, convertedBuildings *geojson.FeatureCollection) {
     // Iterate over each coordinate pair and convert coordinates using convertCoordinates()
     for featureIndex, building := range buildings.Features {
-        fmt.Println("Converting building", building.Attributes.BldNam)
-        for ringIndex, ring := range building.Geometry.Rings {
-            for coordPairIndex, coordpair := range ring {
-                convertedLon, convertedLat := convertCoordinates(coordpair[0], coordpair[1])
-
-                // Reassign coordinate values to the converted coordinates
-                buildings.Features[featureIndex].Geometry.Rings[ringIndex][coordPairIndex][0] = convertedLon
-                buildings.Features[featureIndex].Geometry.Rings[ringIndex][coordPairIndex][1] = convertedLat
+        fmt.Println("Converting building", building.Properties["BldNam"])
+        if building.Geometry.Type == "Polygon" {
```

Comment:
I could, but I'd rather have a separate script for converting those coordinates. I can always move the cs2cs logic into a separate package. I think it would be easier to keep this program limited to pure geojson